Makefile

```diff
@@ -55,7 +55,7 @@ undeploy: manifests
 
 # Generate manifests e.g. CRD, RBAC etc.
 # Requires golang development setup for controller-gen.
-manifests: controller-gen create-installers
+manifests: controller-gen
```

Comment:
But this means we are not generating installers in `make` and `make install` at all ? The idea was to ensure these changes are checked in with PRs.

Reply:
Perhaps we could move what's currently in `manifests` into another target called `create-crds` and then have `manifests` be `manifests: create-crds create-installers`. That way when doing `make manifests` it creates CRDs and installers (as I suppose this change was originally intended to do).

Reply:
yep exactly!

Reply:
didnt want to change the default kubebuilder make targets, created a new target generate_and_install which does as the name says